nav2_semantic_map_server/CMakeLists.txt

```diff
@@ -0,0 +1,31 @@
+cmake_minimum_required(VERSION 3.5)
```

Comment:
What's this package? I see a Dummy map server, should this be used in tests or something?

Reply:
Yes, this is dummy semantic map server made to publish `CostmapFilterInfo` messages required by costmap filters. I've added it to force costmap filters to work well in its current design.

Reply:
Can this be put in the tests directory and named accordingly in the costmap_2d package or nav2_system_tests (if doing full system) then?

Reply:
`CostmapFilterInfo` publisher was moved to `nav2_costmap_2d/test/costmap_filter_info` directory producing `nav2_costmap_2d/test/costmap_filter_info/costmap_filter_info_pub` binary in `build` directory.